### DISCUSSION TITLE
Run MxNet model in C++ with variable input size.

### DISCUSSION THREAD

**Discussion by cao-nv** [apache/incubator-mxnet#20032] [Q&A]
Hi everyone, 
I'm working on a project using MxNet C++. As far as I know, the input size of a MxNet model in C++ must be fixed while generating an `executor`. 

My question is, how to make a model works with variable input images without generating an `executor` for each input. 

Many thanks.

**Comment by szha** [comment]
I think what you are looking for is MXExecutorReshapeEx. See https://github.com/apache/incubator-mxnet/blob/v1.x/include/mxnet/c_api.h#L2529-L2569

**Reply by cao-nv** [reply]
> I think what you are looking for is MXExecutorReshapeEx. See https://github.com/apache/incubator-mxnet/blob/v1.x/include/mxnet/c_api.h#L2529-L2569

Thank you for replying me! 

I took a look at the function `MXExecutorReshapeEx`, but I can not figure out the  difference between using that function and create a new executor.
